hedera-node/hedera-network-admin-service-impl/build.gradle.kts

```diff
@@ -25,6 +25,7 @@ dependencies {
     testImplementation(project(":hedera-node:node-app-service-token"))
     testImplementation(testFixtures(project(":hedera-node:node-config")))
     testImplementation(testFixtures(project(":hedera-node:node-app-spi")))
+    testImplementation(testFixtures(project(":hedera-node:node-app-service-mono")))
```

Comment:
Why is this needed? Can the query handler be implemented entirely based on the modular code paths? Why does it need the mono service test fixtures?

Reply:
I was able to do some refactoring and remove this dependency.